services/frontend-v3/src/components/exFeeder/ExFeeder.jsx

```diff
@@ -4,27 +4,29 @@ import ExFeederView from "./ExFeederView";
 import { ProfileInfoPropType } from "../../utils/customPropTypes";
 import ProfileCards from "../profileCards/ProfileCards";
 
-const ExFeeder = ({ data, type }) => {
+const ExFeeder = ({ data, editableCardBool }) => {
   return (
     <ProfileCards
       titleId={<ExFeederView data={data} />}
       cardName="exFeeder"
       editUrl="/profile/edit/personal-growth?tab=ex-feeder"
       id="card-profile-ex-feeder"
       data={data}
-      type={type}
+      editableCardBool={editableCardBool}
       visibility={data.visibleCards.exFeeder}
+      editableCardBool={editableCardBool}
+      visibility={data.visibleCards.education}
```

Comment:
Wait, you mapped the visibility to the visibleCards.education?

Reply:
fixed i forgot to clean it up